### PR TITLE
Adds dataset page design

### PR DESCRIPTION
**Pull request by maxf**

### CODE REVIEW

File: app/views/dataset/index.html [hunk 1]
```diff
@@ -0,0 +1,160 @@
...
+            <p>Dataset describing the mean number of cats per square kilometre across GB</p>
+          </li>
+          <li>
+            <p><a href="#">Car parks</a></p>
+            <p>Dataset describing the mean number of cats per square kilometre across GB</p>
+          </li>
+        </ul>
+
+
+        <p>In there anything wrong with this dataset?</p>
```
Comment (hannako) [inline app/views/dataset/index.html]: typo :)

Comment (maxf) [reply app/views/dataset/index.html]: Thanks. Will be fixed once I assign this finished PR

File: app/assets/javascripts/application.js [hunk 2]
```diff
@@ -34,8 +34,9 @@ var ShowHide = function() {
 
 ShowHide.prototype = {
   toggle : function(event) {
-    $(event.target).parents(this.selector).find(this.contentSelector).toggle()
-    $('.showHide-control').html(this.isOpen? '-' : '+');
+    var parentShowHide = $(event.target).parents(this.selector)
+    parentShowHide.find(this.contentSelector).toggle()
+    parentShowHide.find(this.controlSelector).html(this.isOpen? '-' : '+');
```
Comment (hannako) [inline app/assets/javascripts/application.js]: this isn't very DRY sorry, let me know how I should improve it!

Comment (maxf) [reply app/assets/javascripts/application.js]: I wouldn't worry about it for now. But I'll take a look tomorrow.

**Review by rossjones — APPROVED** [review]

**Pull request merged**